1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 8 Line 19_
```markdown

In these materials, you will see something like “try the command `$pwd`”. 

Code blocks (like the following) will also be extensively used. Note that you don’t actually type the $ before the command.

```
# This is a comment, the $ on the next line is followed by a command.
$pwd
```
I'd like to not use this `$` syntax, just have the commands bare.

Less confusion, and the Anaconda Prompt that students will be using in this course doesn't use the `$` prefix at all.